bindings/matrix-sdk-ffi/src/ruma.rs

```diff
@@ -1104,6 +1108,63 @@ pub enum AccountDataEvent {
     },
 }
 
+/// The policy that decides if media previews should be shown in the timeline.
+#[derive(Clone, uniffi::Enum, Default)]
+pub enum MediaPreviews {
+    #[default]
+    On,
+    Private,
+    Off,
+}
+
+impl From<RumaMediaPreviews> for MediaPreviews {
+    fn from(value: RumaMediaPreviews) -> Self {
+        match value {
+            RumaMediaPreviews::On => Self::On,
+            RumaMediaPreviews::Private => Self::Private,
+            RumaMediaPreviews::Off => Self::Off,
+            _ => Default::default(),
+        }
+    }
+}
+
+impl From<MediaPreviews> for RumaMediaPreviews {
+    fn from(value: MediaPreviews) -> Self {
+        match value {
+            MediaPreviews::On => Self::On,
+            MediaPreviews::Private => Self::Private,
+            MediaPreviews::Off => Self::Off,
+        }
+    }
+}
+
+/// The policy that decides if avarars should be shown in invite requests.
```

Comment:
Might be nice to elaborate which avatars, the room avatar, the inviter avatar, both?

```suggestion
/// The policy that decides if avatars should be shown in invite requests.
```